ci/docs/build.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+# Copyright (c) 2018, NVIDIA CORPORATION.
+##########################################
+# cuGraph GPU build & testscript for CI  #
+##########################################
+set -e
+
+# Logger function for build status output
+function logger() {
+  echo -e "\n>>>> $@\n"
+}
+
+# Set path, build parallel level, and CUDA version
+export PATH=/conda/bin:/usr/local/cuda/bin:$PATH
+export PARALLEL_LEVEL=4
+export CUDA_REL=${CUDA_VERSION%.*}
+export CUDF_VERSION=0.7.*
+export RMM_VERSION=0.7.*
+
+# Set home to the job's workspace
+export HOME=$WORKSPACE
+
+################################################################################
+# SETUP - Check environment
+################################################################################
+
+logger "Check environment..."
+env
+
+logger "Check GPU usage..."
+nvidia-smi
+
+logger "Activate conda env..."
+source activate gdf
+conda install -c nvidia/label/cuda$CUDA_REL -c rapidsai/label/cuda$CUDA_REL -c rapidsai-nightly/label/cuda$CUDA_REL -c numba -c conda-forge \
+    cudf=$CUDF_VERSION rmm=$RMM_VERSION nvgraph networkx python-louvain sphinx sphinx_rtd_theme \
```

Comment:
@dillon-cullinan and @raydouglass I am still worried about this `nvgraph` dependency I keep seeing around. Recall that the current nvgraph conda package is not going to work for 0.7 release.

Reply:
nvgraph is still listed as a dependency in the conda recipes so that's why is it included here. If it needs to be removed everywhere, can you open an issue? Any dev can make that change.

Reply:
Sure, I commented a week ago on nvgraph open sourcing about that https://github.com/rapidsai/cugraph/pull/194#issuecomment-486789449. I'll create a separate issue if that works better.

Reply:
I created https://github.com/rapidsai/cugraph/issues/266 to keep track of this.